mkdocs.yml

```diff
@@ -32,5 +32,154 @@ extra_javascript:
   # Read the Docs requires JQuery for its JavaScript code to inject the flyout
   # menu. Material for MkDocs does not come with JQuery.
   - "https://code.jquery.com/jquery-3.6.2.min.js"
-#nav:
-#- Welcome: index.md
+nav:
+- Home: index.md
+  - About CivicActions:
+      - Mission and values: 020-about-us/mission-values.md
+      - Culture: 020-about-us/culture.md
+      - Diversity, Equity, Inclusion, and Accessibility (DEIA):
+          - DEIA at CivicActions: 050-how-we-work/diversity-equity-inclusion/diversity-equity-inclusion/purpose.md
+          - Defining DEIA: 050-how-we-work/diversity-equity-inclusion/diversity-equity-inclusion/defining-dei.md
+          - Affinity channels: 050-how-we-work/diversity-equity-inclusion/diversity-equity-inclusion/affinity-channels.md
+          - DEIA - Get involved: 050-how-we-work/diversity-equity-inclusion/diversity-equity-inclusion/deia-get-involved.md
+          - Resources: 050-how-we-work/diversity-equity-inclusion/diversity-equity-inclusion/resources.md
```

Comment:
Looks like these have an extra `diversity-equity-inclusion` folder in the path

Reply:
Fixed